combat/CombatDamage.h

```diff
@@ -0,0 +1,35 @@
+#ifndef _CombatDamage_h_
+#define _CombatDamage_h_
+
+#include "../universe/Condition.h"
+#include "../universe/EnumsFwd.h"
+#include "../universe/Ship.h"
+
+namespace Combat {
+    /** Container for return values of ResolveFighterBouts */
+    struct FighterBoutInfo {
+        struct StateQty {
```

Comment:
is the sub-struct necessary?

Reply:
Probably not I guess, ResolveFighterBouts is old code which I moved from MeterBrowseWnd.